src/containers/Error/index.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
```

Comment:
That is exactly what this component does. Maybe errorTitle isn't exactly the name to use should I use `errorText`?